packaging/creators.json

```diff
@@ -1,113 +1,120 @@
+  },
+  {
+    "displayname": "Abijith Bahuleyan",
+    "authorname": "Bahuleyan, Abijith",
+    "affiliation": "APJ Abdul Kalam Technical University",
+    "github": "twentyse7en",
+    "contribution": "Fix issue 81"
+  },
+  {
+    "displayname": "Jan Caron",
+    "authorname": "Caron, Jan",
+    "affiliation": "Jülich Research Centre",
+    "github": "jan-car",
+    "contribution": "Discussions, advanced color wheel for vector field visualization",
+    "orcid": "0000-0002-0873-889X"
+  },
+  {
+    "displayname": "Rahul Chandra",
+    "authorname": "Chandra, Rahul",
+    "affiliation": "Chandigarh University",
+    "github": "irahulcse",
+    "contribution": "Test and update of examples"
+  },
+  {
+    "displayname": "Magnus Nord",
+    "authorname": "Nord, Magnus",
+    "affiliation": "University of Antwerp",
+    "github": "magnunor",
+    "contribution": "Discussions, tests, code in related projects",
+    "orcid": "0000-0001-7981-5293"
+  },
+  {
+    "displayname": "Colin Ophus",
+    "authorname": "Ophus, Colin",
+    "affiliation": "Lawrence Livermore National Lab",
+    "github": "cophus",
+    "contribution": "Discussions, sample code for K2 reader, overview of related projects, reference files",
+    "orcid": "0000-0003-2348-8558"
+  },
+  {
+    "displayname": "Simon Peter",
+    "authorname": "Peter, Simon",
+    "github": "probonopd",
+    "contribution": "Help with setting up AppImage building and Continuous Integration"
+  },
+  {
+    "displayname": "Jay van Schyndel",
+    "authorname": "Schyndel van, Jay",
+    "affiliation": "Monash University eResearch Centre",
+    "github": "ozej8y",
+    "contribution": "Fix issue 80 to allow container deployment"
+  },
+  {
+    "displayname": "Jaeweon Shin",
+    "authorname": "Shin, Jaeweon",
+    "affiliation": "ETH Zürich",
+    "github": "jaewshin",
+    "contribution": "Single pass numerically stable standard deviation"
+  },
+  {
+    "displayname": "Knut Müller-Caspary",
+    "authorname": "Müller-Caspary, Knut",
+    "affiliation": "Jülich Research Centre",
+    "contribution": "Scientific advisory, discussion, sample data regarding strain mapping, sample code for MIB reader",
+    "orcid": "0000-0002-2588-7993"
+  },
+  {
+    "displayname": "Rafal E. Dunin-Borkowski",
+    "authorname": "Dunin-Borkowski, Rafal E.",
+    "affiliation": "Jülich Research Centre",
+    "contribution": "Scientific advisory, resources, discussion, publications",
+    "orcid": "0000-0001-8082-0647"
+  },
+  {
+    "displayname": "Anand Baburajan",
+    "authorname": "Baburajan, Anand",
+    "affiliation": "APJ Abdul Kalam Technological University",
+    "github": "anandbaburajan",
+    "contribution": "Fixed issue 267"
+  },
+  {
+    "displayname": "Shankhadeep Dey",
+    "authorname": "Dey, Shankhadeep",
+    "affiliation": "Siliguri Institute of Technology",
+    "github": "iamshankhadeep",
+    "contribution": "Added types to libertem.api.Context #643"
+  }
+]
```

Comment:
Thanks for adding yourself to `creators.json`! It is a bit problematic though to re-indent the whole file. This makes it harder for us to review the changes, and may cause merge conflicts for other contributors, or maybe even yourself if you need to rebase this PR to the current master at a later point.

Reply:
Sorry, it was my mistake I did not notice it.